sssom/cli.py

```diff
@@ -627,11 +619,7 @@ def rewire(
     g.parse(input, format=input_format)
     rewire_graph(g, msdf, precedence=precedence)
     rdfstr = g.serialize(format=output_format).decode()
-    if output:
-        with open(output, "w") as stream:
-            stream.write(rdfstr)
-    else:
-        print(rdfstr)
+    print(rdfstr, file=output)
```

Comment:
print lets you say where to print via the `file` argument, use this to deduplicate code like this a ton